### PR TITLE
14148

### PR DESCRIPTION
**Pull request by ketaki-deodhar**
*Issue #:* /bcgov/entity#14148

*Description of changes:*
Insert Court Order and Plan of Arrangement section for ULC application only. Add respective unit tests.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/mixins/filing-template-mixin.ts [hunk 1]
```diff
@@ -200,6 +200,15 @@ export default class FilingTemplateMixin extends DateMixin {
         filing.incorporationApplication.incorporationAgreement = {
           agreementType: this.getIncorporationAgreementStep.agreementType
         }
+
+        if (this.getEntityType === CorpTypeCd.BC_ULC_COMPANY) {
```
Comment (severinbeauvais) [inline src/mixins/filing-template-mixin.ts:204]: I think there's a global/store getter for this already that you can use here, and on line 322.

**Review by severinbeauvais — COMMENTED** [review]
After the other reviewers have looked at this, I'll post my approval.

**Review by lambert-alex — APPROVED** [review]
LGTM

File: src/mixins/filing-template-mixin.ts (hunk 1 above)
Comment (ketaki-deodhar) [reply src/mixins/filing-template-mixin.ts]: Done.

**Review by ChrisGabel23 — APPROVED** [review]

**Review by severinbeauvais — APPROVED** [review]

**Pull request merged**